Python/Product/EnvironmentsList/ToolWindow.xaml

```diff
@@ -206,13 +200,6 @@
                     <RowDefinition Height="*" />
                 </Grid.RowDefinitions>
 
-                <ContentControl Grid.Row="0"
```

Comment:
There is only one item left in the grid. Do you need that grid at all?

Reply:
Don't need it, I can remove it.

Reply:
@AlexanderSher : I removed the grid.